frontend/src/morpheus/containers/OfficePage.js

```diff
@@ -54,33 +84,9 @@ const OfficePage = ({
               history.replace(`/morpheus/office/${room.id}`);
             }}
             onEnterMeeting={(event) => {
-              emitEnterInRoom(room.id);
               onSetCurrentRoom(room);
 
-              if(room.externalMeetUrl){
-                emitStartMeeting();   
-                const externalMeetRoom = window.open(room.externalMeetUrl);
-
-                const externalMeetRoomMonitoring = () => {
-                  window.setTimeout(() => {
-                    if (externalMeetRoom.closed) {
-                      emitLeftMeeting();
-                    }else{
-                      externalMeetRoomMonitoring();
-                    } 
-                  }, 1000);
-                }
-
-                externalMeetRoomMonitoring();
-
-              }else{
-                const redirectUrl = `/morpheus/room/${room.id}`;
-                if (event.ctrlKey) {
-                  window.open(redirectUrl, "_blank");
-                } else {
-                  history.push(redirectUrl);
-                }
-              }
+              enterRoom(room, event, event.ctrlKey);
```

Comment:
You call with params `room, event, event.ctrlKey` and the method receives `room, history, openInNewTab`.

- `event.ctrlKey` is a boolean?
- `event` isn't a history.

Reply:
Hey! Sorry about event, it should be `history` instead. I've fixed it, also extracted `event.ctrlKey` into a variable to better represent its meaning.

Yes, `event.ctrlKey` is a boolean.